pkg/smartcontract/manifest/standard/nep11.go

```diff
@@ -0,0 +1,152 @@
+package standard
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
+)
+
+var nep11Base = &Standard{
+	Manifest: manifest.Manifest{
+		ABI: manifest.ABI{
+			Methods: []manifest.Method{
+				{
+					Name: "balanceOf",
+					Parameters: []manifest.Parameter{
```

Comment:
If we want to add parameters names checks, then we should add them to the standard.

Reply:
Fixed.